zee_slider.js

```diff
@@ -56,6 +56,9 @@
     if (this.options.autoPlay === true) {
       initializeAutoPlay.call(this);
     }
+    if (this.options.showBullets === true) {
+      initializeBulletsLinks.call(this);
```

Comment:
using call here is a bit wired, we can design the class like 
```
const slider =  {
  index: 0,
current:1,
blah: 'sss',
get: function(){
return this.index; /// here you get access to the parent object.
}
}
```

or 
```js
function Slider(){
  this.name = 'zienab';
  this.get = function(){
	return this.name;
  }
}
var slider = new Slider();
slider.get() // zienab
```

Reply:
@Mohamed-amin  I didn't get this part. :confused:

Reply:
```js
initializeBulletsLinks()
```
this func depends on other members of the slider class.
so you can make it part of the class it self.

Reply:
@Mohamed-amin Still not there yet. I will give it a try and push it to a separate commit.